be/src/olap/types.h

```diff
@@ -568,6 +577,13 @@ struct FieldTypeTraits<OLAP_FIELD_TYPE_CHAR> : public BaseFieldtypeTraits<OLAP_F
         auto slice = reinterpret_cast<const Slice*>(data);
         return HashUtil::hash(slice->data, slice->size, seed);
     }
+    static char* allocate_value_from_arena(Arena* arena) {
```

Comment:
Now, Field has already contained the length of column.
I think it's more suitable to put this function in Filed other than here

Reply:
agree;Field.h can decide allocate type value whether with size or not, but now ColumnZoneMapBuilder's field doesn't contains whole column info;
I think we can init a Field in SegmentWriter.init ,and then transfer the field to ColumnWriter and ColumnZonemap to instead of TypeInfo, but this need to change ColumnWriter constructor and ColumnZonemap, and change corresponding UT
So I prepare to submit another pr to fix the problem that string's ColumnZoneMap's max/min value init with real length.
Currently we first use fixed length.

Reply:
OK, please don't forget delete this function then.

Reply:
👌